src/core/mdm-catalogue-item.model.ts

```diff
@@ -151,3 +152,54 @@ export type PathableDomainType =
   | 'userImageFiles'
   | 'versionedFolders'
   | 'versionLinks';
+
```

Comment:
I was surprised we didn't have these things already defined for other facet operations